pull.py

```diff
@@ -57,8 +57,40 @@
         'FDLv1.2',
         'FDLv1.3',
     ],
+    'FreeArt': [ # any version
+        'LAL-1.2',
+        'LAL-1.3',
+    ],
     'FreeBSDDL': ['FreeBSD'],  # unify (multi-tag)
-    # FIXME: still working through this
+    'NPL': [ #any version
+        'NPL-1.0',
+        'NPL-1.1',
+    ],
+    'OSL': [ # any version through 3.0
+        'OSL-1.0',
+        'OSL-1.1',
+        'OSL-2.0',
+        'OSL-2.1',
+        'OSL-3.0',
+    ],
+    'RPL': [ # any version - Note that FSF website does not state any version, but references version 1.3 in the URL.  It is assumed that it also covers version 1.1 and 1.5, but this should be verified with FSF.
```

Comment:
The FSF at least occasionally says “any version” when that's their intention.  Until we get clarity on this, can we drop it from this PR (and the associated `RPL-*` `IDENTIFIERS` entries) and come back to them in follow-up work?